core/web-assets/src/main/assets/js/apps/onms-graph/index.js

```diff
@@ -144,7 +144,6 @@ const getDimensionsForElement = (el, def) => {
 };
 
 const render = () => {
-  console.log('GraphContainer: rendering'); // eslint-disable-line no-console
```

Comment:
Can we restore the console logs for when the graphs do get rendered?

Reply:
Now we can add log based on  graph name. Is that okay since it could be lot. The other way is to count the total and log once everything gets rendered but that won't be solve for the filtering case.  I would prefer logging for each graph based on graph name.

Reply:
Update logger with graph name